public/app/plugins/datasource/dashboard/DashboardQueryEditor.tsx

```diff
@@ -131,19 +156,45 @@ export function DashboardQueryEditor({ panelData, queries, onChange, onRunQuerie
           value={selected}
           onChange={(item) => onPanelChanged(item.value!)}
         />
-      </InlineField>
-
-      {results && !loadingResults && (
-        <div className={styles.results}>
-          {query.panelId && (
-            <VerticalGroup spacing="sm">
-              {results.map((target, i) => (
-                <DashboardQueryRow editURL={editURL} target={target} key={`DashboardQueryRow-${i}`} />
-              ))}
-            </VerticalGroup>
+      </Field>
+
+      {loadingResults ? (
+        <Spinner />
+      ) : (
+        <>
+          {results && Boolean(results.length) && (
+            <Field label="Queries">
+              <VerticalGroup spacing="sm">
+                {results.map((target, i) => (
+                  <div className={styles.queryEditorRowHeader} key={`DashboardQueryRow-${i}`}>
+                    <div>
+                      <img src={target.img} width={16} />
+                      <span className={styles.refId}>{`${target.refId}:`}</span>
```

Comment:
```suggestion
                      <span className={styles.refId}>{target.refId}:</span>
```

Same no? :) just nitpick